src/osp/Resource/blueprints.h

```diff
@@ -26,15 +26,14 @@
 
 #include "../types.h"
 
-#include "Resource.h"
-#include "PrototypePart.h"
+#include "resources.h"
 
 namespace osp
 {
 
 struct BlueprintMachine
 {
-    NodeMap_t m_config;
+    //NodeMap_t m_config;
```

Comment:
remove

Reply:
There's a chance that NodeMap_t will be typedefed to something else eventually, though I'm not sure what yet. Similar case to other comment above